README.md

```
@@ -2,9 +2,10 @@
 
 Marinade's Stake Auction for Solana validators.
 
-<a href="https://www.npmjs.com/package/@marinade.finance/ds-sam-sdk">
-  <img src="https://img.shields.io/npm/v/%40marinade.finance%2Fds-sam-sdk?logo=npm&color=377CC0" />
-</a>
+|Package       | NPM registry |
+| -------------| ------------ |
+| ds-sam-sdk   | <a href="https://www.npmjs.com/package/@marinade.finance/ds-sam-sdk"><img src="https://img.shields.io/npm/v/%40marinade.finance%2Fds-sam-sdk?logo=npm&color=377CC0" /></a> |
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)